packages/core/src/common/idle.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable } from 'inversify';
+import { Event } from '../common/event';
```

Comment:
the file should be named after the main exported type

Reply:
Why? Since when do we have this policy?

Reply:
i think it always was, please check the coding guideline, it should be somewhere there

Reply:
https://github.com/theia-ide/theia/wiki/Coding-Guidelines#names:
> Name files after the main Type it exports

Reply:
Ohh, come on... I will change if you feel better:

```
 - promise-util.ts
 - cli.ts
 - extension-protocol.ts
 - ...
```

Reply:
we don't always pay attention, it is true, also utils and protocol are special since there is no a main concept